bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,81 @@
+# ABB/Busch-free@home Smart Home binding
+
+ openHAB ABB/Busch-free@home binding based on the offical free@home local api
+
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/free_at_home_logo_1.jpg)
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/abb_freeathome_2_0.png)
+# Description
+
+This openHAB binding allows you to connect your free@home Smart Home system from ABB / Busch-Jaeger to openHAB and to control and observe most of the components.
+It requires a System Access Point with version 2.6.1 or higher.
+
+# Supported Devices
+
+- ABB / Busch-Jaeger System Access Point 2.0
+- free@home Switch Actuator Sensor 1/1, 2/1, 2/2 (wired and wireless)
+- free@home Dimming Actuator Sensor 1/1, 2/1 (wired and wireless)
+- free@home Blind Actuator Sensor 1/1, 2/1 (wired and wireless)
+- free@home Movement Detector Actuator
+- free@home Radiator Thermostat
+- free@home Switch Actuator 4-channel
+- free@home Switch Actuator 4-channel
+- free@home Dimming Actuator 4- and 6-channel
+- IP-touch panel (function: door opener, door ring sensor)
+- Hue devices (untested)
+
+
+# Tested SysAP Versions
+
+|Version|Supported|
+|---|---|
+|2.6.1+|yes|
+
+
+# Setup / Installation
+
+## Prerequisites
+
+To make use of this Binding first the local free@home API has to be activated. The API is disabled by default!
+
+1. Open the free@home next app
+2. Browse to "Settings -> free@home settings -> local API and activate the checkbox
+
+## Discovery
+
+The free@home bridge shall be added manually. Once it is added as a Thing with correct credentials, the scan of free@home devices will be possible.
+The devices with multiple channels are devided into multiple devices with a single channel. The label of these devices is built-up with the same deivce name and free@home device ID but the channel number is mentioned in the Thing label as well for better identification.
+
+## Setup
+
+1. Enter your openHAB webfrontend with `<device IP>:8080`
+2. Log into openHAB with your crendetials at the lower left side
+3. Browse to "Settings -> Things" and press the "+" symbol
+4. Choose "FreeAtHome System Binding" and click "Free@home Bridge"
+5. Add the required data: SysAP IP address, username and password
+**ATTENTION:** The username here has to be from "Settings -> free@home settings -> local API, NOT the username from webfrontend or used in the app for login)
+
+6. Press save in the righter upper corner
+7. If everthing is right the Bridge should went "Online"
+8. "Scan" for the free@home devices and set up the your free@home components as Thing
```

Comment:
The readme shouldn't reference any UI related functions to prevent the readme becoming obsolete when the UI changes.

Reply:
UI reference is removed